test/clj/chronograph/domain/acl_test.clj

```diff
@@ -20,9 +20,9 @@
             {user-id :users/id} (factories/create-user)
             {organization-id :organizations/id} (factories/create-organization admin-id)]
         (with-transaction [tx db/datasource]
-          (acl/create! tx {:user-id user-id
-                           :organization-id organization-id
-                           :role acl/admin}))
+          (acl/create! tx {:acls/user-id user-id
+                           :acls/organization-id organization-id
+                           :acls/role acl/admin}))
```

Comment:
Minor nit. Less line noise if we namespace-qualify the whole map, like in line 26 below.